Sources/SMTP/Client/SMTPClient+Convenience.swift

```diff
@@ -1,14 +1,16 @@
 import Transport
+import Sockets
 
-extension SMTPClient {
+extension SMTPClient where StreamType == TCPInternetSocket {
```

Comment:
do we need this constraint? can the initializer just be `try StreamType(scheme: ` etc. in the implementations